examples/workflows/plot_simulate_beta.py

```diff
@@ -129,7 +129,7 @@ def add_beta_drives(net, beta_start):
         'beta_dist', tstart=beta_start, tstart_std=0., tstop=beta_start + 50.,
         burst_rate=1., burst_std=10., numspikes=2, spike_isi=10, n_drive_cells=10,
         location='distal', weights_ampa=weights_ampa_d1,
-        synaptic_delays=syn_delays_d1, seedcore=2)
+        synaptic_delays=syn_delays_d1, seedcore=20)
```

Comment:
Glad you were able to find the right seeds to address this. Perhaps this example is something we should discuss during the retreat too w.r.t. how to explain in the text the importance/implication of the random seeding.

Reply:
Totally agree!